### PR TITLE
Automatically run db:migrate when releasing to Heroku

### PR DESCRIPTION
**Pull request by tvararu**
There's no reason we shouldn't use this on Heroku, since it isn't production data anyway.

### CODE REVIEW

**Review by misaka — APPROVED** [review]

**Comment by tvararu** [comment]
Review app appears to not be deploying correctly. Closing this for now as it's a rabbit hole.

**Review by misaka — APPROVED** [review]
Let's give it a go?

**Comment by tvararu** [comment]
@misaka had to modify this to get it to work on review apps. Check it out now.

File: db/migrate/20230113195033_update_children_to_latest.rb [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+
     # Do not run this migration on production or on existing data. It is a
     # destructive migration. Remove child records before running.
-    raise "Child records exist, cannot run migration" if Child.any?
+    if ActiveRecord::Base
+      .connection
+      .execute("SELECT EXISTS (SELECT 1 FROM children)")
+      .first["exists"]
+      raise "Child records exist, cannot run migration"
+    end
```
Comment (misaka) [inline db/migrate/20230113195033_update_children_to_latest.rb:12]: Can we just remove this migration? Or do we actually still need it?

Comment (tvararu) [reply db/migrate/20230113195033_update_children_to_latest.rb]: I've removed it!

**Pull request merged**